live_pipeline/requirements.txt

```diff
@@ -1,7 +1,9 @@
-requests
-pytest
-requests-mock
 boto3
-python-dotenv
+ipykernel 
```

Comment:
I don't think this is required. The Docker image won't be using Jupyter Notebooks so we probably don't want to install the library